rust/otel-arrow-rust/src/encode/record/logs.rs

```diff
@@ -142,6 +142,12 @@ impl LogsRecordBatchBuilder {
         self.observed_time_unix_nano.append_value(&val);
     }
 
+    /// append a value to the `observed_time_unix_nano` array `n` times
+    pub fn append_observed_time_unix_nano_n(&mut self, val: Option<i64>, n: usize) {
+        let val = val.unwrap_or(0);
```

Comment:
Could we simply use `i64` as the value argument type instead of using `Option<i64>`?